src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -9882,4 +9950,85 @@ public enum GCGlobalMechanisms
         Demotion = 0x8,
         CardBundles = 0x10,
     }
+
+    #region private types
+    /// <summary>
+    /// ClrTraceEventParserState holds all information that is shared among all events that is
+    /// needed to decode Clr events.   This class is registered with the source so that it will be
+    /// persisted.  Things in here include
+    /// 
+    ///     * TypeID to TypeName mapping, 
+    /// </summary>
+    // [SecuritySafeCritical]
+    internal class ClrTraceEventParserState : IFastSerializable
+    {
+        internal void SetTypeIDToName(int processID, Address typeId, long timeQPC, string typeName)
+        {
+            if (_typeIDToName == null)
+                _typeIDToName = new HistoryDictionary<string>(500);
+
+            _typeIDToName.Add(typeId + ((ulong)processID << 48), timeQPC, typeName);
+        }
+
+        internal string TypeIDToName(int processID, Address typeId, long timeQPC)
+        {
+            lazyTypeIDToName.FinishRead();      // We don't read fileIDToName from the disk unless we need to, check
```

Comment:
:bulb: Comment references wrong field name. Also consider placing the comment on the line before the code so it's easier to be consistent across code formatters.